### ISSUE TITLE
pre-training

### ISSUE THREAD

**Issue by ghost** [fmahoudeau/MiCT-Net-PyTorch#7]
Excuse me, I want to do pre-training on the kinetics700, but I don't know how to do it, can you give me some advice? I haven't found a similar pre-training code for reference. Do you know if there is one?

**Comment by fmahoudeau** [comment]
You could start with ucf101.py as template, but reducing data augmentation to horizontal flipping and corner cropping. I would start with Kinetics400. Have you downloaded the dataset already ?

**Comment by ghost** [comment]
> You could start with ucf101.py as template, but reducing data augmentation to horizontal flipping and corner cropping. I would start with Kinetics400. Have you downloaded the dataset already ?

Thank you for your reply. The training set and verification set of kinetics400 data set have been downloaded, and the whole data set of kinetics700 has been downloaded. Unfortunately, there is no test set of kinetics400 data set.

**Comment by fmahoudeau** [comment]
> > You could start with ucf101.py as template, but reducing data augmentation to horizontal flipping and corner cropping. I would start with Kinetics400. Have you downloaded the dataset already ?
> 
> Thank you for your reply. The training set and verification set of kinetics400 data set have been downloaded, and the whole data set of kinetics700 has been downloaded. Unfortunately, there is no test set of kinetics400 data set.

How is the training going ?
